pkg/reconciler/sinkbinding/controller.go

```diff
@@ -161,6 +180,24 @@ func (s *SinkBindingSubResourcesReconciler) Reconcile(ctx context.Context, b psb
 			Name:       sb.Spec.Sink.Ref.Name,
 		}, b)
 	}
+	
+	featureFlags := feature.FromContext(ctx)
```

Comment:
You probably need to add the featurestore before that to the context, or add it to the `SinkBindingSubResourcesReconciler` and get it from there.

Reply:
PTAL now.